package/wpe/wpewebkit/Config.in

```diff
@@ -122,12 +122,34 @@ endchoice
 
 if !BR2_PACKAGE_WPEWEBKIT_DEBUG
 
-config BR2_PACKAGE_WPEWEBKIT_SYMBOLS
-	bool "add symbols to a release build"
-        default y if BR2_PACKAGE_WPEWEBKIT_DEBUG
-        default n
-	help
-	  Add symbols to a release build
+choice
+        bool "Symbols level"
+        default BR2_PACKAGE_WPEWEBKIT_SYMBOLS_LEVEL_2 if BR2_PACKAGE_WPEWEBKIT_DEBUG
+        default BR2_PACKAGE_WPEWEBKIT_SYMBOLS_NO
+        help
+          Choose symbols level to apply to the compiler. WebKit 2.38 might not build with level 2 or 3.
```

Comment:
IMO, the amount of debuginfo added by the compiler should be independent from `BR2_PACKAGE_WPEWEBKIT_DEBUG`, which would make possible the following interesting options:

- Picking a debug build with minimal info (`-g1`)
- Picking a debug build build without symbols (`-g0`), which will still have assertions and a few other debug facilities built into WebKit.

Let's remove the `if ! BR2_PACKAGE_WPEWEBKIT_DEBUG` conditional in line 123.